app/src/main/java/com/infomaniak/drive/MainApplication.kt

```diff
@@ -207,4 +196,18 @@ class MainApplication : Application(), ImageLoaderFactory, DefaultLifecycleObser
 
         override fun getCurrentUserId(): Int = AccountUtils.currentUserId
     }
+
+    /**
+     * Reasons to discard Sentry events :
+     * - Application is in Debug mode
+     * - User deactivated Sentry tracking in DataManagement settings
+     * - The exception was an [ApiController.NetworkException], and we don't want to send them to Sentry
+     */
+    private fun configSentry() {
+        this.configureSentry(
```

Comment:
Add `this` is not required
```suggestion
        configureSentry(
```